packages/squiggle-lang/src/rescript/Distributions/PointSetDist/PointSetDist_Scoring.res

```diff
@@ -14,3 +14,33 @@ module KLDivergence = {
       quot < 0.0 ? Error(Operation.ComplexNumberError) : Ok(-.answerElement *. logFn(quot))
     }
 }
+
+module LogScoreWithPointResolution = {
+  let logFn = Js.Math.log
+  let score = (
+    ~priorPdf: option<float => float>,
```

Comment:
Why was it important to have these be in float => float formats, instead of just the Continuous dist?

Reply:
I thought it was more consistent with the theme of the file: in the `KLDivergence` module, we elicit the underlying points from the dists and operate in this file on them. That's why I chose to have the caller extract out the `pdf` before calling the function.